CODE_OF_CONDUCT.md

```diff
@@ -1,3 +1,21 @@
-# Code of Conduct
+# Conda Organization Code of Conduct
 
```

Comment:
Thoughts on moving the link to the top and into an admonition?

```suggestion

> **Note**
> Below is the short version of our CoC, see the long version [here](https://github.com/conda-incubator/governance/blob/main/CODE_OF_CONDUCT.md).

```

Reply:
Good idea.